### ISSUE TITLE
restart review of agency workflows to loop out core team

### ISSUE THREAD

**Issue by gbinal** [18F/api.data.gov#301]
- [ ] Broadcasting Board of Governors
  - [x] embed signup form
  - [ ] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] BusinessUSA.gov API
  - [x] embed signup form
  - [ ] change contact email address on [confirmation page](http://business.usa.gov/developer/api-signup)
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] Department of Commerce
  - [x] embed signup form
  - [ ] change contact email address on [confirmation page](https://www.commerce.gov/page/api-documentation-commercegov)
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] Federal Communications Commission
  - [ ] [embed signup form](https://ach.fcc.gov/for-developers/)
  - [ ] change contact email address on confirmation page when it exists
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] Federal Election Commission
  - [ ] [embed signup form](https://api.open.fec.gov/developers)
  - [ ] change contact email address on confirmation page when it exists
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] Food and Drug Administration
  - [x] embed signup form
  - [x] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] General Services Administration - Auctions
  - [x] embed signup form
  - [ ] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [x] ~~General Services Administration - Data.gov~~  
- [ ] General Services Administration - FBOpen 
  - [ ] [embed signup form](https://pages.18f.gov/fbopen/apidocs/)
  - [ ] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] General Services Administration - SAM
  - [x] embed signup form
  - [ ] change contact email address on [confirmation page](https://gsa.github.io/sam_api/sam/key)
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] General Services Administration - SFTools
  - [ ] [embed signup form](https://sftool.gov/Developers/Overview)
  - [ ] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] NASA
  - [x] embed signup form
  - [x] change contact email address on [confirmation page](https://api.nasa.gov/index.html#apply-for-an-api-key)
  - [x] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] National Institutes of Health
  - [x] embed signup form
  - [ ] change contact email address on [confirmation page](http://niaid.github.io/3dpx_api/api_key/)
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] National Renewable Energy Laboratory
  - [x] embed signup form
  - [x] change contact email address on [confirmation page](https://developer.nrel.gov/signup/)
  - [x] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] Regulations.gov API
  - [x] embed signup form
  - [ ] change contact email address on [confirmation page](https://regulationsgov.github.io/developers/key/)
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] U.S. Department of Agriculture - ERS
  - [ ] embed [signup form](http://www.ers.usda.gov/developer/)
  - [ ] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] U.S. Department of Agriculture - NNS
  - [x] embed signup form  - __note need to change example url__
  - [ ] change contact email address on [confirmation page](http://ndb.nal.usda.gov/ndb/doc/#)
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email
- [ ] U.S. Department of Education
  - [ ] embed [signup form](https://github.com/18F/open-data-maker/blob/api-docs/API.md)
  - [ ] change contact email address on confirmation page
  - [ ] change contact email address in confirmation email
  - [ ] change reply-to email address in confirmation email

**Comment by GUI** [comment]
@gbinal: The recommended instructions on https://github.com/18F/api.data.gov/wiki/User-Manual:-Agencies#linking-to-your-own-contactsupport-address should be updated to also include customizing the e-mail subject and from address (the e-mail will still technically be from "noreply@api.data.gov", but the sender's name will be adjusted).

To verify the instructions, I went ahead and ran through them for Regulations.gov, since I knew they already had a preferred contact form. I think everything checks out, and I see no references to api.data.gov anywhere (except in the API url, but that's to be expected, since they're using our domain). I already adjusted their API backends, we'll just need to get this pull request accepted and then their website will be updated and we can check them off: https://github.com/regulationsgov/developers/pull/56
